Comments/ClientApp/src/components/Tutorial/Tutorial.js

```diff
@@ -24,7 +24,9 @@ export class Tutorial extends Component<PropsType, StateType> {
 	}
 
 	setCookie = (isPanelVisible: boolean) => {
-		Cookies.set("TutorialVisible", isPanelVisible);
+		if(typeof window.CookieControl !== "undefined" && window.CookieControl.getCategoryConsent(0)){
```

Comment:
This bit in the docs was outdated. It's going to be `CookieControl.preferenceCookies` now. Sorry.
```suggestion
		if(CookieControl.preferenceCookies){
```

We've also changed the way the script loads to not need to check typeof

Reply:
Thanks Ian. the new syntax is much nicer.
I've tweaked the code in another commit. would you mind taking a look please?